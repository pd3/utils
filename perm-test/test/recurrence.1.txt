

    12345678901234567890
         bb   bb   b
        bbb  b  b     b
       bb      bb   bb
        bb
       BBBB  BBBB  BBBB
       4567  0123  6789
         ttttt  tttt
        .cc cc. ooo ...
     xx. ccc c.. ooo ..x
      x.. ccc ..o oo. .xx
       ..c ccc .oo o.. 


    ./recurrence-test -c test/recurrence.1.calls.txt -a test/recurrence.1.bg.txt -l test/recurrence.1.tgt.txt -f test/recurrence.1.fai -n 1e5 -s 1587471914
        TEST gene_A  2  84629 15371 0
        TEST gene_B  0  0     44601 55399
    
        A exp: (7/18)**2   = 0.1512
        B exp: (12./18)**2 = 0.4444


