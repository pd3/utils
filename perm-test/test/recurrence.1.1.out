SEED	1536695456
NITER	1.000000e+05
MSG	Maximum chromosome randomization error due to length discretization: 0.0e+00%
# TEST:
#     - gene .. the region label
#     - nObserved .. number of hits observed in input data in this gene
#     - nSimFewer .. number of simulations with fewer hits in the gene than observed in the data
#     - nSimEqual .. as above, but with the same number of hits than observed
#     - nSimExceeded .. as above, but with the more hits than observed
# DIST:
#     - gene
#     - nSimHits .. the number of simulations with 0,1,2,etc. hits in the gene
# [1]TEST	[2]gene	[3]nObserved	[4]nSimFewer	[5]nSimEqual	[6]nSimExceeded
TEST	gene_A	2	84629	15371	0
TEST	gene_B	0	0	44601	55399
# [1]DIST	[2]gene	[3-]nSimHits (0,1,etc. hits)
DIST	gene_A	37182	47447	15371
DIST	gene_B	44601	44263	11136
